instrumentation/rocketmq-client-4.8/javaagent/rocketmq-client-4.8-javaagent.gradle

```diff
@@ -4,7 +4,7 @@ muzzle {
   pass {
     group = "org.apache.rocketmq"
     module = 'rocketmq-client'
-    versions = "[4.8.0,)"
+    versions = "[4.0.0,)"
```

Comment:
@addname if tests pass against 4.0, let's run tests against that version and rename the module to `rocketmq-client-4.0`

Reply:
nevermind, i took a quick look and tests didn't compile against earlier versions, not a problem to keep as-is